ansible_collections/arista/avd/plugins/plugin_utils/avdfacts.py

```diff
@@ -8,6 +8,19 @@ def __init__(self, hostvars, templar):
         self._hostvars = hostvars
         self._templar = templar
 
+    @classmethod
+    def __keys(cls):  # pylint: disable=bad-option-value, unused-private-member # CH Sep-22: Some pylint bug.
```

Comment:
Yes it is pylint version dependent. So it varies a lot with different environments. Since ours / ansible's minimum requirements do not include the fix for the bug(s) I had to disable it.